aiida/restapi/translator/base.py

```diff
@@ -347,12 +271,12 @@ def set_query(
         nalist=None,
```

Comment:
What is now the interaction between `alist`, `nalist`, and `attributes_filter`? Shouldn't at least one of this go away or all possible combinations make sense?

Reply:
The filters `alist`, `nalist` are used in the `/nodes/<id>/contents/attributes/` endpoint, while the `attributes_filter` is used for the `/nodes` and `/nodes/<id>` endpoint.

Reply:
To do:
1. Double check we are never using nalist in the frontend
2. drop `nalist` and `nelist`
3. rename `alist` to `attributes_filter`, `elist` to `extras_filter`